packages/components/app/styles/components/code-block/index.scss

```diff
@@ -0,0 +1,254 @@
+  }
+
+  code,
+  pre {
+    display: inline;
+    font-size: 0.9em; // as discussed with designers, we reduce the size for optical/visual balance
+    font-family: var(--token-typography-code-100-font-family);
+    line-height: 1em;
+  }
+
+  // Default styling for bare HTML links not using HDS::Link components
+  a:not([class*="hds-"]) {
+    color: var(--code-block-color-foreground-action-default);
+
+    &:focus,
+    &:focus-visible {
+      text-decoration: none;
+      outline: 2px solid var(--token-color-focus-action-internal);
+      outline-offset: 1px;
+    }
+
+    &:hover {
+      color: var(--token-color-foreground-action-hover);
+    }
+
+    &:active {
+      color: var(--token-color-foreground-action-active);
+    }
+  }
+}
+
+.hds-code-block__body {
+  position: relative;
+}
+
+// Code
+.hds-code-block__code {
+  display: block;
+  margin: 0;
+  padding: $code-block-code-padding;
+  overflow: auto;
+}
+
+// CopyButton
+.hds-code-block__copy-button {
+  // Overriding default colors
+  color: var(--code-block-color-foreground-primary);
+  background-color: var(--code-block-color-surface-faint);
+  border: 1px solid var(--code-block-color-border-strong);
+
+  &:hover,
+  &.mock-hover {
+    background-color: var(--code-block-color-surface-primary);
+    border-color: var(--code-block-color-border-strong);
+  }
+
+  &:active,
+  &.mock-active {
+    background-color: var(--code-block-color-surface-interactive-active);
+  }
+
+  &:focus,
+  &.mock-focus,
+  &:focus-visible {
+    background-color: var(--code-block-color-surface-faint);
+  }
+
+  &.hds-copy-button--status-success {
+    .flight-icon {
+      color: var(--code-block-color-foreground-success);
+    }
+  }
+
+  &.hds-copy-button--status-error {
+    .flight-icon {
+      color: var(--code-block-color-foreground-critical);
+    }
+  }
+
+  .flight-icon { color: var(--code-block-color-foreground-primary); }
+}
+
+// VARIANTS
+
+// isRounded
+.hds-code-block--is-rounded {
+  border-radius: 6px;
+}
+
+// hasCopyButton
+.hds-code-block--has-copy-button {
+  .hds-code-block__copy-button {
+    position: absolute;
+    top: 8px;
+    right: 8px;
+  }
+}
+
+// LineWrapping
+.hds-code-block--has-line-wrapping {
+  pre,
+  code { white-space: pre-wrap; }
+}
+
+// Linenumbers
+// Note: Prism.js is using the specific class name "line-numbers" to determine implementation of line numbers in the UI
+.hds-code-block {
+  // LineNumbers plugin styles ---------------
+  &.line-numbers {
+    counter-reset: linenumber;
+
+    .hds-code-block__code {
+      position: relative;
+      // reserve space for line numbers
+      padding-left: calc(#{$code-block-line-numbers-width} + #{$code-block-code-padding});
+    }
+
+    .line-numbers-rows {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: $code-block-line-numbers-width;
+      padding: $code-block-code-padding 0;
+      background-color: var(--code-block-color-surface-primary);
```

Comment:
Since is the same color as the parent container, is this needed?
```suggestion
```

Reply:
I thought we needed this for specificity, but doesn't seem so. Removed in 636ebad35